### PR TITLE
Add html, css and a readme files and a linter for HTML and CSS

### PR DESCRIPTION
**Pull request by SFarida**
_This pull request is a fix of the initial pull request I created #1. I had to create a new pull request because I had to change the branch name and renaming a branch automatically cancels the pull request._

Add project files alongside a linter for HTML and CSS. The following are included:

- Added a .gitignore file with node_modules, and test.me.
- Added a linter for HTML and CSS.
- Added a README file together with an html and a css file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,277 @@
 # Hello-Microverse
 Apply GitHub flow best practices and provide a good documentation
```
Comment (rskinnerc) [inline README.md]: Hey there @SFarida good job with your project so far 😃😃. 

[OPTIONAL] Please consider removing this line. Following the recommended template would make look your project more professional.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,277 @@
...
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+
+</div>
```
Comment (rskinnerc) [inline README.md]: [OPTIONAL] Please consider removing the default Microverse logo or updating it to your custom logo. 🖼️

File: README.md [hunk 3]
```diff
@@ -1,2 +1,277 @@
...
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
```
Comment (rskinnerc) [inline README.md]: ♻️ Awesome job adding the missing sections to the README file as requested by the previous CR ✌️✌️. However, there are still some changes that need your attention.

- [ ] The main technologies for this project would be HTML and CSS. Please update the technologies list accordingly. You can also add the linters. 🤖

File: README.md [hunk 4]
```diff
@@ -1,2 +1,277 @@
...
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
```
Comment (rskinnerc) [inline README.md]: ♻️ Great job. Your README file looks more professional with the added sections. However, those sections need a little bit more customization. Customizing to your needs/info makes your repo even more professional.

- [ ] Please change _Author1_ to your name.
- [ ] You added the required sections to the README. However, customizing them as per the project features would be awesome. Kindly update so the users have a better picture of how to use this repo (i.e. in the getting started section, explain how to clone the repo, how to start the project and use it) 📋📋📋

File: README.md [hunk 5]
```diff
@@ -1,2 +1,277 @@
...
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (rskinnerc) [inline README.md]: ♻️ Amazing! You added a license to your project 🔏🔏. However, seems like it is pointing to a broken link. 

- [ ] Please fix the path and add the license file you want to use. [This template](https://github.com/microverseinc/readme-template) contains an MIT license template that you can make use of. Kindly update it accordingly.

File: .github/workflows/linters.yml [hunk 6]
```diff
@@ -0,0 +1,56 @@
+name: Linters
```
Comment (rskinnerc) [inline .github/workflows/linters.yml:1]: [OPTIONAL] Even though this is an optional (but highly recommended) change, bear in mind that for **every future project** you will need to set up the linters.

You have done great work setting up GitHub Actions (this file) 💪💪💪. But as part of the good practices and setting up the linters, you must install and configure (add config files) the linters to your project. Please check [this guide on how to set up your HTML/CSS Linters.](https://github.com/microverseinc/linters-config/tree/master/html-css)

File: index.html [hunk 7]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
```
Comment (rskinnerc) [inline index.html:8]: [OPTIONAL] Please consider adding a `title` to your project. Making this habit would make you a better dev. 😃😃😃

**Review by rskinnerc — CHANGES_REQUESTED** [review]
Hi @SFarida  👋,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

giphy


 ## Highlights
- Good job adding the big `node_modules` to `.gitignore`
- Awesome! You set up GitHub actions rightly

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- 🔴 Dear @SFarida please note that you don't need to create separate pull requests whenever a code reviewer asks you for changes. You can keep working on the same branch and push commits to that branch. The PR gets updated automatically. Having said that, this is a Re-review of the #1 Pull Request previously submitted.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SFarida** [comment]
Hello @rskinnerc , in one of the required changes I was suppose to rename my branch. Renaming a branch closes the pull request, I just had to create another one. I am not sure of what to do in such a case now.

**Review by brainconnect93 — APPROVED** [review]
### Hi @SFarida, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 
- Files properly ignored  🥇 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**